src/spec.tex

```diff
@@ -1487,6 +1521,8 @@ \subsection{Handouts}
 \end{document} %!hide
 \end{example}
 
+%$ Paragraph needs some work; unclear what is being said.
+%$ You only get the <mode> slides when producing output of <mode>?
 Because the overlay specification is often used to provide contents for a
 single mode, there exist another way of specifying it for that purpose---if you
 provide only the mode name, this will be the same as setting all other modes to
```

Comment:
```suggestion
Because the overlay specification is often used to provide contents for a
single mode, there exists a short cut: f you
provide only the mode name, this will be the same as setting all other modes to
```